project.clj

```diff
@@ -1,8 +1,7 @@
-(defproject google-civic "2.0.2-SNAPSHOT"
+(defproject google-civic "2.1.0-SNAPSHOT"
   :description "Google Civic Info API client"
   :dependencies [[org.clojure/clojure "1.7.0"]
-                 [clj-http "2.0.0"]
-                 [cheshire "5.5.0"]
-                 [slingshot "0.12.2"]]
+                 [clj-http "2.3.0"]
```

Comment:
But it looks like there have been concurrent releases on the 2.X.X branch along with the same ones of the 3.X.X branch? That implied to me that it was still known to be unstable.